docroot/themes/humsci/humsci_basic/src/js/shared/timeline/expand-collapse-timeline.js

```diff
@@ -19,46 +19,11 @@
         items.forEach((item) => {
           item.removeAttribute('open');
         });
-
-        // Find the summary element and update the aria attribute values
-        const summaries = timeline.querySelectorAll(
-          '.hb-timeline-item__summary',
-        );
-
-        summaries.forEach((summary) => {
-          summary.setAttribute('aria-expanded', 'false');
-          summary.setAttribute('aria-pressed', 'false');
-        });
       });
 
-      // When a user clicks on a timeline, update the aria properties accordingly
+      // When a user clicks on a timeline, set open property accordingly
```

Comment:
This comment is misleading, please remove it. The `open` property is not add via javascript, the browser adds it automatically.